lib/smart_proxy_openscap/foreman_forwarder.rb

```diff
@@ -2,11 +2,11 @@ module Proxy::OpenSCAP
   class ForemanForwarder < Proxy::HttpRequest::ForemanRequest
     include ::Proxy::Log
 
-    def post_arf_report(cname, policy_id, date, data)
+    def post_arf_report(cname, policy_id, date, data, timeout)
```

Comment:
this is also used athttps://github.com/theforeman/smart_proxy_openscap/blob/master/lib/smart_proxy_openscap/openscap_import_api.rb#L15

either make timeout optional or change every call